vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -4804,9 +4805,22 @@ public void setTooltipGenerator(
         this.dataCommunicator.reset();
     }
 
+    /**
+     * Sets the tooltip position relative to the cell that is being hovered or
+     * focused. The default position is {@link TooltipPosition#BOTTOM}.
+     *
+     * @param position
+     *            the position to set
+     */
+    public void setTooltipPosition(TooltipPosition position) {
```

Comment:
Would it make sense to add a getter for consistency?

Reply:
Added a getter. To facilitate this, added a method to the enum to get an enum value from the position string, which returns the bottom position by default.